custom/icds_reports/ucr/reports/mpr/mpr_4b_infra_forms_v2.json

```diff
@@ -144,7 +144,12 @@
         "display": "use_salt"
       }
     ],
-    "sort_expression": [],
+    "sort_expression": [
+      {
+        "field": "month",
+        "order": "DESC"
+      }
+    ],
```

Comment:
out of curiosity, why does this need a sort applied?

Reply:
I commented about this on the ticket.
I think it was worth having this sorted so that the recent month is on top.